momentum/index.d.ts

```diff
@@ -36,6 +36,77 @@ interface MapData {
 	credits: { type: string, user: { alias: string } }[];
 }
 
+ /***************************************************************************************** */
+
+  interface Region extends JsonObject {
+	points: Vec2D[];
+	bottom: number;
+	height: number;
+	teleportTargetName: string; // mutually exclusive to other two teleport fields
```

Comment:
This should be `teleDestTargetName` though really it could become `teleportTargetName` like you have here. Whatever we use should probably match the pos/yaw fields as well (which I still need to add back to game code, so no names there currently.)

Reply:
I've made this change, but let me know if the fields are what you had in mind

Reply:
Sure, seems good